backend/src/ml_space_lambda/utils/iam_manager.py

```diff
@@ -41,6 +41,14 @@ def __init__(self, iam_client=None, sts_client=None):
         self.aws_partition = boto3.Session().get_partition_for_region(boto3.Session().region_name)
         self.iam_client = iam_client if iam_client else boto3.client("iam", config=retry_config)
 
+        env_variables = get_environment_variables()
+        self.data_bucket = env_variables["DATA_BUCKET"]
+        self.system_tag = env_variables["SYSTEM_TAG"]
+        self.notebook_role_name = os.getenv("NOTEBOOK_ROLE_NAME", "")
+        self.default_notebook_role_policy_arns = []
+        self.permissions_boundary_arn = os.getenv("PERMISSIONS_BOUNDARY_ARN", "")
+        self.aws_account = self.sts_client.get_caller_identity()["Account"]
```

Comment:
Is there a reason you're not using the `account_utils` you added for this? Same for the enums.

Reply:
Did this before creating the `account_utils`. Good callout